src/fondant/executor.py

```diff
@@ -33,7 +35,32 @@
 
 
 class Executor(t.Generic[Component]):
-    """An executor executes a Component."""
+    """
+    An executor executes a Component.
+
+    Args:
```

Comment:
Thanks for adding this! 
- Can you indent? 
- No need to add the datatype between two brackets. The convention we follow is to just have it in the function arguments